dev_tools/rubocop/rspec-base.yml

```diff
@@ -0,0 +1,11 @@
+require:
+- rubocop-rspec
```

Comment:
I don't know why [yaml.org](https://yaml.org/) uses a mix of indentation but can you indent all the YAML lists like in the `Exclude` further down?